logistics_planning_base/models/logistics_schedule.py

```diff
@@ -158,6 +159,15 @@ class LogisticsSchedule(models.Model):
         Tecnical field indicating whether this record can be set to done state
         """,
     )
+    etd_date = fields.Date(
+        string="ETD",
```

Comment:
Keep better complete field name in definition, and put `string="ETD"` only in XML views.
```suggestion
        string="ETD Date",
```

Reply:
Still missing the second part (and put string="ETD" only in XML views.)

Reply:
Only in tree view?

Reply:
In both tree & form. Original complete field name will be only available when inspecting fields model structure